test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smppy.py

```diff
@@ -36,7 +36,7 @@
 
 def _skip_if_image_is_not_compatible_with_smppy(image_uri):
     _, framework_version = get_framework_and_version_from_tag(image_uri)
-    compatible_versions = SpecifierSet("==1.13.*")
+    compatible_versions = SpecifierSet("==1.13.*,==2.0.*")
```

Comment:
This doesn't work. Please change this to
```suggestion
    compatible_versions = SpecifierSet(">=1.13.*")
```